sw/device/lib/meson.build

```diff
@@ -118,3 +118,16 @@ sw_lib_usb = declare_dependency(
     ]
   )
 )
+
+# Message pring library (sw_lib_log)
```

Comment:
`# Logging library that prints to UART directly.`

Reply:
Done.